.github/workflows/conformance.yml

```diff
@@ -29,27 +29,27 @@ jobs:
           - kind
           - eks
         config:
-          - name: "consul@v1.11 + consul-k8s@v0.49.0"
-            api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
-            consul-image: "hashicorp/consul:1.11"
-            envoy-image: "envoyproxy/envoy:v1.20-latest"
-            consul-k8s-version: "v0.49.0"
-          - name: "consul@v1.12 + consul-k8s@v0.49.0"
+          - name: "consul@v1.12 + consul-k8s@v0.49.1"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
             consul-image: "hashicorp/consul:1.12"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
-            consul-k8s-version: "v0.49.0"
-          - name: "consul@v1.13 + consul-k8s@v0.49.0"
+            consul-k8s-version: "v0.49.1"
+          - name: "consul@v1.13 + consul-k8s@v0.49.1"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
             consul-image: "hashicorp/consul:1.13"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
-            consul-k8s-version: "v0.49.0"
-          - name: "consul@v1.14-dev + consul-k8s@v0.49.0"
+            consul-k8s-version: "v0.49.1"
+          - name: "consul@v1.14-dev + consul-k8s@v1.0.0"
+            api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
+            consul-image: "hashicorp/consul:1.14"
+            envoy-image: "envoyproxy/envoy:v1.22-latest"
+            consul-k8s-version: "main"
+          - name: "consul@v1.15-dev + consul-k8s@v1.0.0"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
-            consul-image: "hashicorppreview/consul:1.14-dev"
+            consul-image: "hashicorppreview/consul:1.15-dev"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
```

Comment:
Just heads up for documentation purposes to remind us that we need to fix the way this gets passed eventually. Currently with the 1.0.0 chart this is doing nothing, since the `global.imageEnvoy` flag changed to `apiGateway.imageEnvoy` and is [1.23.1](https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/values.yaml#L2856) by default. Since we're still passing `global.imageEnvoy` we're just getting the default in the 1.0.0 CI runs.

Reply:
I think that should be a straightforward enough change.

Reply:
Done in https://github.com/hashicorp/consul-api-gateway/pull/458